### ISSUE TITLE
Implicit dimension choice for softmax has been deprecated. Change the call to include dim=X as an argument.

### ISSUE THREAD

**Issue by santhoshdc1590** [amdegroot/ssd.pytorch#114]
I'm getting this UserWarning:

> Implicit dimension choice for softmax has been deprecated. Change the call to include dim=X as an argument.
> self.softmax(conf.view(-1, self.num_classes)),  # conf preds

**Comment by amdegroot** [comment]
I think you need to pull the latest version.  I fixed these warnings recently, I believe.
